README.md

```diff
@@ -119,37 +119,76 @@ returns
 }
 ```
 
-### Create Event
-POST /api/events/
-```
-{
-  "name": "<event_name>"
-}
-```
+### Get all tasks for a user
+GET /api/tasks/me
+
 returns
 ```
 {
   "data": {
-    "id": <event_id>,
-    "created_at": <timestamp>
-    "name": "<event_name>"
+    "not_scheduled": [
+      {
+        "id": <task_id>,
+        "name": "<task_name>",
+        "description": "<task_description>",
+        "duration": <task_duration_in_seconds>
+        "scheduled": <boolean>,
+        "completed": <boolean>,
+        "user_id": <TWAIN_USER_ID>
+      },
+      ...
+    ], 
+    "scheduled": [
+      {
+        "id": <task_id>,
+        "name": "<task_name>",
+        "description": "<task_description>",
+        "duration": <task_duration_in_seconds>
+        "scheduled": <boolean>,
+        "completed": <boolean>,
+        "user_id": <TWAIN_USER_ID>
```

Comment:
Please make these structures match the most up to date task model above. There are a ton more fields. You can just copy and paste. Also please remove all console.log